Functions/fn_CalculateOverdueFees.sql

```diff
@@ -0,0 +1,29 @@
+	CREATE FUNCTION fn_CalculateOverdueFees (@LoanID int)	
+	RETURNS int AS
+	BEGIN
+		DECLARE @DueDate date;
+		SET @DueDate = (SELECT DueDate FROM Loan WHERE LoanID = @LoanID);
+
+		DECLARE @DateReturned date;
+		SET @DateReturned = (SELECT DateReturned FROM Loan WHERE LoanID = @LoanID);
```

Comment:
you can get the DueDate , DateReturned in the same query :
`SELECT @var1 = column1, @var2 = column2 FROM table_name WHERE condition;
`